src/metaschema/oscal_control-common_metaschema.xml

```diff
@@ -75,12 +76,12 @@
             &allowed-values-control-group-property-name;
                         </allowed-values>
                         <allowed-values target="prop[has-oscal-namespace('http://csrc.nist.gov/ns/rmf')]/@name">
-                                <enum xmlns="https://csrc.nist.gov/ns/rmf" value="method">Type of The Risk Management Framework 800-53A method to be use for assessment.</enum>
+                                <enum value="method">Type of the Risk Management Framework 800-53A method to be used for assessment.</enum>
                         </allowed-values>
                 </constraint>
                 <remarks>
                         <p>A <code>part</code> provides for logical partitioning of prose, and can be thought of as a grouping structure (e.g., section). A <code>part</code> can have child parts allowing for arbitrary nesting of prose content (e.g., statement hierarchy). A <code>part</code> can contain <code>prop</code> objects that allow for enriching prose text with structured name/value information.</p>
-                        <p>A <code>part</code> can be assigned an optional <code>id</code>, which allows for internal and external references to the textual concept contained within a <code>part</code>. A <code>id</code> provides a means for an OSCAL profile, or a higher layer OSCAL model to reference a specific part within a <code>catalog</code>. For example, an <code>id</code> can be used to reference or to make modifications to a control statement in a profile.</p>
+                        <p>A <code>part</code> can be assigned an <code>id</code>, which allows for internal and external references to the part, considered as an entity or object (for example, a discrete item or subitem within a control statement). For example, an OSCAL profile may address parts by their ids in order to identify where controls are to be tailored.</p>
```

Comment:
<p>A <code>part</code> can be assigned an optional <code>id</code>, which allows references to this part from within a catalog, or within an instance of another OSCAL model that has a need to reference the part. Examples of where part referencing is used in OSCAL includes:</p>
<ul>
<li>Referencing a part by id to tailor (make modifications to) a control statement in a profile.</li>
<li>Referencing a control statement represented by a part in a system security plan implemented-requirement where a statement-level response is desired.</li>
</ul>